packages/ListBox/stories/WithTags/examples/AllOptionsAreSelected.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import ListBox, { useListBoxWithTags } from "../../../src/WithTags";
+import animals from "../mocks";
+
+// defaultData and filteredData have the same size.
+const defaultFilteredData = animals.slice(0, 20);
+const defaultData = defaultFilteredData;
+
+export default function App() {
+  const { isSelected, filteredData, getSelectedOptions, ...moreUseListBoxWithTagsProps } = useListBoxWithTags({
+    key: "label",
+    defaultData,
+    defaultFilteredData,
+    defaultSelectedKeys: ["Alpaca", "Ant", "Ape", "Armadillo"],
+  });
+
+  return (
+    <div style={{ padding: "32px" }}>
+      <ListBox
+        allOptionsAreSelected
```

Comment:
So if all options are selected, it shows the message even when I haven't expanded the ListBox?  That's kinda weird.

![Screen Shot 2020-12-18 at 10 26 37 AM](https://user-images.githubusercontent.com/23224777/102648029-a00c1180-411b-11eb-9afd-d425acd4c6cc.png)

Reply:
isn't this the example that @sunminsarahlee shared with you? if you don't use allOptionAreSelected option, still render the filter and shows you the legend that there are not results.

Reply:
also if this is now what she wants we can modify it once she is back from 🌴 @sunminsarahlee 😄